behaviors/magic-button.md

```diff
@@ -47,4 +51,18 @@ url: $SITE_PREFIX (this is a ~ special token ~ that evaluates to the prefix of c
 property: url
 ```
 
+### (ﾉ◕ヮ◕)ﾉ*:・ﾟ✧ "grab the image url from a lede component, then ask mediaplay for the caption"
+
+```yaml
+component: feature-lede
+transform: getComponentInstance
+url: $SITE_PREFIX
+property: imgUrl
+moreMagic:
+  -
+    transform: mediaplayUrl (to change the image url into a string we can query mediaplay's api with)
+    url: [mediaplay api url]
+    property: metadata.caption
+```
```

Comment:
The common case is only doing one iteration of magic in a button, so it's easier to surface those things the way I did. Plus, the properties allowed in moreMagic are a lot more restrictive.